samples/text/text.cpp

```diff
@@ -1228,8 +1228,18 @@ MyPanel::MyPanel( wxFrame *frame, int x, int y, int w, int h )
     m_textrich->AppendText("This text should be cyan on blue\n");
     m_textrich->SetDefaultStyle(wxTextAttr(*wxBLUE, *wxWHITE));
     m_textrich->AppendText("And this should be in blue and the text you "
-                           "type should be in blue as well");
-
+                           "type should be in blue as well.\n");
+    m_textrich->SetDefaultStyle( wxTextAttr() );
+    m_textrich->AppendText(wxT("And there is a "));
+    wxTextAttr attr = m_textrich->GetDefaultStyle();
+    attr.SetFontUnderline( TRUE, wxTEXT_ATTR_UNDERLINE_WAVE, *wxRED );
```

Comment:
Capitalized `TRUE` is/was Windows-specific and is not a boolean but an integer. It is also used in the gtk code.

Reply:
Will fix in the next commit.